src/smc-webapp/course/actions.ts

```diff
@@ -775,8 +773,11 @@ export class CourseActions extends Actions<CourseState> {
     if (store == undefined) {
       return;
     }
-    const students = store.get_students().valueSeq().toArray();
-    await amap(students, PARALLEL_LIMIT, this._delete_student)
+    const students = store
```

Comment:
Interesting -- I wonder if these formatting changes are due to a prettier update, or if we committed code that wasn't formatted before.  Nothing for you to do.